Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "lurk"
-version = "0.2.0"
-authors = ["porcuquine <porcuquine@gmail.com>"]
```

Comment:
😢

Reply:
We can do a concatenation if you want :)